admittance-control/admittance_control.script

```diff
@@ -138,3 +243,5 @@ end
 
 # Run the admittance control function
 admittance_control(compliance_vector = [0, 0, 1, 0, 0, 1])
+
+admittance_control(1,1,1,1,1,1,1)
```

Comment:
This this function call is misleading. Suggest we only have one call of the function at then of the file

Reply:
That makes sense